content/test/integration/syncronization/name-change-handling.spec.ts

```diff
@@ -0,0 +1,101 @@
+import ms from "ms"
+import fs from "fs"
+import { Timestamp } from "@katalyst/content/service/time/TimeSorting"
+import { DAOClient } from "decentraland-katalyst-commons/src/DAOClient"
+import { Environment, EnvironmentConfig } from "@katalyst/content/Environment"
+import { TestServer } from "../TestServer"
+import { buildDeployData, buildBaseEnv, deleteServerStorage, buildDeployDataAfterEntity, stopServers } from "../E2ETestUtils"
+import { assertHistoryOnServerHasEvents, buildEvent } from "../E2EAssertions"
+import { MockedDAOClient } from "./clients/MockedDAOClient"
+import { delay } from "decentraland-katalyst-commons/src/util"
+import { ControllerEntity } from "@katalyst/content/controller/Controller"
+import { DeploymentEvent } from "@katalyst/content/service/history/HistoryManager"
+import { EntityType } from "@katalyst/content/service/Entity"
+
+
+describe("End 2 end - Name change handling", function() {
+
+    const SYNC_INTERVAL: number = ms("1s")
+    const DAO_SYNC_INTERVAL: number = ms("8s")
+    let jasmine_default_timeout
+    let server1: TestServer, server2: TestServer
+    let dao
+
+    beforeAll(() => {
+        jasmine_default_timeout = jasmine.DEFAULT_TIMEOUT_INTERVAL
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000
+    })
+
+    afterAll(() => {
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = jasmine_default_timeout
+    })
+
+    beforeEach(async () => {
+        dao = MockedDAOClient.withAddresses('http://localhost:6060', 'http://localhost:7070')
+        server1 = await buildServer("Server1_", 6060, SYNC_INTERVAL, dao)
+        server2 = await buildServer("Server2_", 7070, SYNC_INTERVAL, dao)
+    })
+
+    afterEach(async () => {
+        await stopServers(server1, server2)
+        deleteServerStorage(server1, server2)
+    })
+
+    it('When a node starts, it get all the previous history', async () => {
```

Comment:
should we add a test to validate the recurrent synchronization too? I'm assuming this test only checks the startup.